serialosc/index.md

```diff
@@ -27,34 +27,42 @@ The rest of this document details what these pre-made libraries abstract, for th
 
 `serialoscd` keeps watch for connection and disconnection of devices. When connected, a new process is launched 'serialosc-device' which communicates on its own port.
 
-We will assume a theoretical application on port 66666 on localhost is trying to connect to serialosc.
+We will assume a theoretical application on port 66666 on localhost is trying to connect to serialosc. We'll also assume you have `liblo` installed, which includes `oscsend` and `oscdump`.
 
-To list the available devices, query `serialoscd` which listens on port 12002:
+First, connect your grid to your computer. Then, open a terminal window and set up an OSC monitor:
 
-- to port 12002: `/serialosc/list localhost 66666`
+- `oscdump 6666`
+
+Open another terminal window. To list the available devices, we'll query `serialoscd` which listens on port 12002, and we'll route the response to our monitor:
+
+- `oscsend localhost 12002 /serialosc/list si localhost 6666`
 
 Say we have an 8x16 grid attached, results might look like this:
 
-- received on port 66666: /serialosc/device "m1000000" "monome 128" 21008
+- `e83daeee.d2ea63b5 /serialosc/device ssi "m46674021" "monome 128" 17675`
 
-Here, port 21008 is where the device is listening. To have this grid send to our application:
+Here, port 17675 is where the device is listening. To have this grid send to our application:
 
-- to port 21008: `/sys/port 66666`
+- `oscsend localhost 17675 /sys/port i 6666`
 
 Now when keys on the grid are pressed we should see something like:
 
 ```
-/grid/key 0 0 1
-/grid/key 0 2 1
+<timestamp> /monome/grid/key iii 9 3 1
+<timestamp> /monome/grid/key iii 9 3 0
 ```
 
-...which follow the form `/prefix/key x y z`. Note, the "prefix" is changeable with a message.
+...which follow the form `/prefix/grid/key x y z`. Note, that while the "prefix" defaults to `/monome`, it's changeable with any string. For example, to change our prefix to `/blinky`:
+
+- `oscsend localhost 17675 /sys/prefix s "/blinky"`
+
+Note that when a grid is disconnected + reconnected, its prefix will return to `/monome`.
```

Comment:
i thought this was actually saved in the serialosc prefs and would be restored

Reply:
oh! yeah idk why my initial disconnect/reconnect came back as `/monome`, but a re-test proves that. sorry for the noise, will fix!